single_pages/dashboard/store/products.php

```diff
@@ -1177,45 +1177,59 @@ class="fa fa-pencil"></i></a>
+                <?php if($tColumn == "pVariations" || $tColumn == "selectPageTemplate"){ ?>
+                  <input type="hidden" name="column[<?= $tColumn ?>]" value="<?= strlen($tDesc['default'])==0 ? "" : $tDesc['default']  ?>"/>
+                <?php  }else{ ?>
+                  <tr>
+                    <td><label><?= t($tDesc['label']) ?></label><?php echo $tDesc['label']=="In Product Groups" ? " (Separated by comma) " : ""; ?> <em><?= strlen($tDesc['default'])==0 ? "" : "(Default: ".$tDesc['default'].")"   ?></em></td>
 
-                  <td style="width: 200px;">
-                    <select class="form-control" name="column[<?= $tColumn ?>]">
-                      <option value="" selected>Select Column</option>
-                      <?php
-                      $ctr = 0;
+                    <td style="width: 200px;">
 
-                        foreach($headers as $head){?>
-                          <option value="<?= $ctr++ ?>"><?= $head ?></option>
-                        <?php }?>
-                    </select>
-                  </td>
-                </tr>
+                      <select class="form-control" name="column[<?= $tColumn ?>]">
+                        <option value="" selected>Select Column</option>
+                        <?php
+                        $ctr = 0;
+
+                          foreach($headers as $head){?>
+                            <option value="<?= $ctr++ ?>"><?= $head ?></option>
+                          <?php }?>
+                      </select>
+                    </td>
+                  </tr>
+                <?php } ?>
               <?php endforeach; ?>
+        <?php if($area != "Options" && $area != "Detail Page") {?>
             </tbody>
           </table>
         </div>
+        <?php } ?>
       <?php endforeach; ?>
       <div class="form-group">
         <input type="button" class="btn btn-primary import-submit-btn btn ccm-input-submit" data-action="<?= $view->action('beginImport')?>" id="importbutton" name="importbutton" value="<?= t('Begin Import');?>"/>
       </div>
       <?php }else{ ?>
-
+      <?php
+      	$pk=Package::getByHandle('community_store');
+      	$ppath=$pk->getRelativePath();
+      ?>
       <div class="form-group">
           <div class="ccm-system-errors alert alert-danger alert-dismissable import-error" style="display:none;"><?php echo t('Please upload a valid CSV file!'); ?></div>
 
           <label class="control-label"><?php echo t('Upload CSV file'); ?></label>
           <div><input type="file" name="csv" id="csv"></div>
+
+
       </div>
       <div class="form-group">
         <input type="button" class="btn btn-primary import-submit-btn btn ccm-input-submit" id="upload" name="upload" value="<?= t('Upload');?>"/>
-
+        <span id="loading_img" style="display: none;"><img src="<?php echo $ppath; ?>/images/gif-load.gif" /> <?php echo t('Uploading CSV file...'); ?></span>
```

Comment:
Instead of adding a file in here, I'd suggest just using a font-awesome spinner, i.e. replace the <img with:

<i class="fa fa-spinner fa-spin"></i>

There there doesn't have to be another file added